app_grim_reefer/sample.yaml

```diff
@@ -0,0 +1,16 @@
+# This file is provided so that the application can be compiled using Twister,
+# the Zephyr testing tool. In this file, multiple combinations can be specified,
+# so that you can easily test all of them locally or in CI.
+sample:
+  description: Example application
+  name: example-application 
+common:
+  build_only: true
+  integration_platforms:
+    - custom_plank
+    - nucleo_f302r8
```

Comment:
```suggestion
    - grim_reefer
```

Reply:
fixed in [6cf8032](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/6cf803229d38690de83d6db332228645b0495534)